cpp/include/gpu_operations.h

```diff
@@ -47,6 +56,54 @@ class GpuOperations {
   static Matrix<T> OuterProduct(const Vector<T> &a, const Vector<T> &b);
 };
 
+template <typename T>
+Matrix<T> GpuOperations<T>::Multiply(const Matrix<T> &a, const Matrix<T> &b) {
+  if (a.cols() == b.rows()) {
+  int m = a.rows();
```

Comment:
We don't need to, because in that cmake file the wildcard is used. Back to this case, I found that the cpplint cannot detect the indentation problem even if I set up the framework from scratch (Like clone a new repository, run cmake from the beginning)